allure-commandline/src/main/java/ru/yandex/qatools/allure/command/AbstractCommand.java

```diff
@@ -0,0 +1,73 @@
+package ru.yandex.qatools.allure.command;
+
+import io.airlift.command.Option;
+import io.airlift.command.OptionType;
+import org.apache.log4j.Level;
```

Comment:
couse I set up slf4j with log4j way: 
http://saltnlight5.blogspot.ru/2013/08/how-to-configure-slf4j-with-different.html